src/components/facet-browser/helper.ts

```diff
@@ -132,8 +134,29 @@ export const getFacetFieldTranslation = (name: FacetField) => {
   }
 };
 
+export const createFacetsMap = (
+  facets: FacetResult[]
+): { [key: string]: FacetValue[] } => {
+  return facets.reduce(
+    (acc: { [key: string]: FacetValue[] }, facet: FacetResult) => {
+      const newAcc = { ...acc };
+
+      facet.values.forEach((value) => {
+        const combinedKey = `${facet.name}:${value.key}`;
+        newAcc[combinedKey] = newAcc[combinedKey]
+          ? [...newAcc[combinedKey], value]
+          : [value];
```

Comment:
Isn't it possible just to write:
```typescript
newAcc[combinedKey] = value;
```
If you overwrite it does not matter because the new object should be the same as the former.